internal/services/network/application_gateway_resource_test.go

```diff
@@ -7662,6 +7672,12 @@ resource "azurerm_application_gateway" "test" {
     priority                   = 10
   }
 
+  ssl_policy {
+    policy_type          = "Custom"
+    min_protocol_version = "TLSv1_1"
```

Comment:
The minimum protocol version default will now be `TLSv1_2` so can we update the checks in the test instead of the config. It's a better practice for the tests to verify that current security standards/defaults are working in the API than to ensure outdated security defaults and practices still work.

Reply:
updated